pulpcore/app/serializers/user.py

```diff
@@ -234,7 +234,10 @@ class UserRoleSerializer(ModelSerializer, NestedHyperlinkedModelSerializer):
     )
 
     content_object = ContentObjectField(
-        help_text=_("Optional pulp_href of the object the permissions are to be asserted on."),
+        help_text=_(
+            "Optional pulp_href of the object the permissions are to be asserted on. "
+            "Set to 'null' when assigning model-level permissions."
```

Comment:
I'd like to get a native speaker involved here. It turned out that what was here was not even obvious to people i had explained it before. I'm still going to leave some suggestions:

```suggestion
            "The pulp_href of the object the permissions are to be asserted on. "
            "Set to 'null' for assigning model-level roles."
```
```suggestion
            "The pulp_href of the object the roles permissions are to be asserted on. "
            "If set to 'null', permissions will act on the model-level."
```

rationale:
- avoid "optional", it was too confusing
- assignment happens for roles, checking is done by permission

Reply:
@mdellweg I agree this is a difficult thing to explain.  The new (suggested) phrasing sounds fine to me.  A slightly simpler variant might be:

> The pulp_href of the object for which role permissions should be asserted. If set to 'null', permissions will act on the model-level.

Does that sound reasonable?  This is subtle and I'm not as familiar with RBAC so I could be missing a small detail.